.chloggen/httpcheck-parse-scheme.yaml

```diff
@@ -0,0 +1,24 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I think bug_fix is probably fine, as it was not working before w/o the scheme

Reply:
I'm just a bit worried that for a customer that had their collector configured with a couple different receivers including the HTTP Check receiver, they might have not even noticed that the HTTP Check receiver is not working. After this change, when they upgrade they'll see the collector failing to start and might be baffled, not seeing any breaking changes in the Changelog. What do you think?

Reply:
Either i think is fine, but i'm ok if you want to rename this a breaking change

Reply:
Changed to `breaking`, I think it's better. This component is in `development` status, so introducing breaking changes in it shouldn't be a big deal.